flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -154,12 +175,15 @@ function set_conf_ssl {
        rm -rf "${TEST_DATA_DIR}/ssl"
     fi
     mkdir -p "${TEST_DATA_DIR}/ssl"
+
     NODENAME=`hostname -f`
     SANSTRING="dns:${NODENAME}"
-    for NODEIP in `hostname -I | cut -d' ' -f1` ; do
+    for NODEIP in $(get_node_ip) ; do
```

Comment:
Why did you remove the additional `| cut -d' ' -f1`?

Reply:
`hostname -I ` would return a list of IP addresses separated by a space, and the cut would only select one of those. The intended behaviour according to my discussion with @NicoK is that we append all IP addresses returned by `hostname -I` to the SANSTRING, which is why I removed that line

Reply:
Thanks for the clarification.